src/components/productList/utils/filters.js

```diff
@@ -18,9 +18,9 @@ const category = [
   },
   {
     id: 3,
-    name: "T-shirts",
-    value: "tees",
-    label: "T-shirts",
+    name: "Mug",
+    value: "mug",
+    label: "Mug",
```

Comment:
Why are you using two different attributes with same value?

Reply:
Thanks, Manoj For pointing out that will resolve it.